crates/context_aware_config/src/api/default_config/handlers.rs

```diff
@@ -189,9 +189,7 @@ async fn create(
         AppHeader::XConfigVersion.to_string(),
         version_id.to_string(),
     ));
-    Ok(http_resp.json(json!({
-        "message": "DefaultConfig created/updated successfully."
-    })))
+    Ok(http_resp.json(json!(default_config)))
```

Comment:
```suggestion
    Ok(http_resp.json(default_config))
```

json! macro is not needed here, http_resp.json function as well as `Json` type of actix_web by default serializes its data

Reply:
Nice! I didn't know that